specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-03-01/storagecache.json

```diff
@@ -279,16 +279,13 @@
         ],
         "responses": {
           "200": {
-            "description": "Cache deleted.",
-            "schema": {}
+            "description": "Cache deleted."
```

Comment:
See above comments (in the examples context) regarding 200 OK for async/long-running ops.